addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6818,7 +6818,18 @@ msgctxt "#13516"
 msgid "Add art"
 msgstr ""
 
-#empty strings from id 13517 to 13549
+#: system/settings/settings.xml
+msgctxt "#13517"
+msgid "Threshold when atempo resampling kicks in"
```

Comment:
ok, title text: "Threshold for pitch correction"

help text: "When the speed change exceeds this threshold, a pitch-correction filter will be applied. This avoids the "chipmonk voices" that normally result from speeding up a video."